app/pug/common.pug

```diff
@@ -1,49 +1,17 @@
 
+block pugscripts
+  - function isAuthenticated()     { return !!(user); }
+  - function user_email(aUser)     { if (isAuthenticated()) return ((aUser.emails || [])[0] || {}).value || ""; else return "[Please login to see!]" };
+  - function user_shortname(aUser) { if (isAuthenticated()) return aUser.displayName || user_email(aUser); else return "[Please login to see!]" };
```

Comment:
So by that you mean the syntax is `if (condition) return [value1]; return [value2]`?

Reply:
Yup!

Reply:
Huh ... even after 6 months JavaScript continues to feel weird to me!

Reply:
The same applies in C too.  The statement after the condition executes in all cases, except for the condition because there is a return statement inside of it.